browser/brave_wallet/brave_wallet_service_unittest.cc

```diff
@@ -137,6 +145,18 @@ class BraveWalletServiceUnitTest : public testing::Test {
     ASSERT_TRUE(callback_called);
     ASSERT_EQ(token2_->symbol, "UNI");
 
+    callback_called = false;
+    mojom::ERCTokenPtr erc721_token;
+    GetRegistry()->GetTokenByContract(
+        "0x06012c8cf97BEaD5deAe237070F9587f8E7A266d",
+        base::BindLambdaForTesting([&](mojom::ERCTokenPtr token) {
+          erc721_token_ = std::move(token);
+          callback_called = true;
+        }));
+    base::RunLoop().RunUntilIdle();
```

Comment:
not a big deal but I think it's preferred to be done with `base::RunLoop` instead of `callback_called` and `RunUntilIdle`. I think because `RunUntilIdle` is easier to get broken like in a case where an extra callback chain is added.